code/game/objects/items/weapons/shields.dm

```diff
@@ -133,10 +133,16 @@
 	flags_equip_slot = ITEM_SLOT_BACK
 	max_integrity = 300
 	soft_armor = list("melee" = 50, "bullet" = 50, "laser" = 0, "energy" = 100, "bomb" = 30, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 35)
-	hard_armor = list("melee" = 0, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hard_armor = list("melee" = 5, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	force = 20
 	slowdown = 0.2
 
+/obj/item/weapon/shield/riot/marine/mob_can_equip(mob/M, slot, warning = TRUE)
+	if(slot != SLOT_L_HAND && slot != SLOT_R_HAND && CHECK_BITFIELD(flags_item, NODROP)) //Let the player know why they can't move the shield to non-hand slots
+		to_chat(M, "<span class='notice'>We must first loosen [src]'s straps!</span>")
+
+	return ..()
+
```

Comment:
```suggestion

```

Reply:
Better to keep this in order to let the shield user know why he can't equip the shield to his back, though I'll use the syntax you used earlier.